core/context.js

```diff
@@ -101,6 +103,10 @@ class TelegrafContext {
     return this.update.callback_query
   }
 
+  get passportData () {
+    return this.update.passport_data
```

Comment:
`passport_data` is a part of `Message`, not `Update`:

`return this.message && this.message.passport_data`

Reply:
Good catch, thanks!